projects/systelab-components/src/lib/datepicker/README.md

```diff
@@ -27,7 +27,7 @@ All the input parameters are optional.
 | warnDaysAfter | number | | If set, when a date is selected the input background is set to red when that date happens after than the system's current date plus the number of days specified. |
 | autofocus | boolean | false | If true the input is automatically focused |
 | fromDateForRelativeDates | date | undefined | Date to be used as from date to calculate the new date when using the shortcuts (ie. 3d)
-| showDateFormatOnError | boolean | false | If true, it will show a error message on placeholder when `error` property is true. Also, you can customize error message by adding `BAD_DATE_FORMAT` bundle key to your translation service file. If there's no `BAD_DATE_FORMAT` key, it only will show expected date format.
+| showDateFormatOnError | boolean | false | If true, it will show a error message on placeholder when `error` property is true.
```

Comment:
"When the date is invalid, a red border is displayed on input" is more accurate.

In addition, the attribute "showDateFormatOnError" does not longer apply. It may be "showError"